app.ts

```diff
@@ -0,0 +1,187 @@
+*/
+function onloadValues() {
+	let toValue = (<HTMLInputElement>document.getElementById("toValue")).value;
+	let fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value;
+
+	from = 0;
+	fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value = from.toString();
+	let rowsFittable = amountOfRowsThatFit();
+	to = from + rowsFittable;
+	toValue = (<HTMLInputElement>document.getElementById("toValue")).value = to.toString();
+	callHeaders(from, to);
+}
+
+/**
+ *Adjust the values in the start and end boxes according to the screen heigth whenever it resizes
+ */
+function fromAndToValuesAdjustment() {
+	let toValue = (<HTMLInputElement>document.getElementById("toValue")).value;
+	let fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value;
+
+	from = parseInt(fromValue);
+	let rowsThatFit = amountOfRowsThatFit();
+	to = from + rowsThatFit;
+
+	if (to > 100000) {
+		window.alert("Nee, stout >.<,your search has exceeded the maximum rows")
+		from = 100000 - rowsThatFit;
+		to = 100000;
+	} else if (from < 0) {
+		window.alert("Nee, stout >.<,your search has exceeded the minimum rows")
+		from = 0;
+		to = from + rowsThatFit;
+	}
+	fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value = from.toString();
+	toValue = (<HTMLInputElement>document.getElementById("toValue")).value = to.toString();
+	callHeaders(from, to);
+
+}
+/**
+ *Executes when the user presses the next button
+ */
+function pressedNext() {
+
+	let toValue = (<HTMLInputElement>document.getElementById("toValue")).value;
+	let fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value;
+
+	let rowsThatFit = amountOfRowsThatFit();
+
+	from = parseInt(toValue) + 1;
+	to = from + rowsThatFit;
+
+	if (to > 100000) {
+		window.alert("Nee, stout >.<,your search has exceeded the maximum rows")
+		from = 100000 - rowsThatFit;
+
+		to = 100000;
+	}
+	fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value = from.toString();
+	toValue = (<HTMLInputElement>document.getElementById("toValue")).value = to.toString();
+
+
+	callHeaders(from, to);
+}
+/**
+ *Executes when the user presses the previous button
+ */
+function pressedPrevious() {
+
+	let toValue = (<HTMLInputElement>document.getElementById("toValue")).value;
+	let fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value;
+	let rowsThatFit = amountOfRowsThatFit();
+
+	to = from - 1;
+	from = parseInt(fromValue) - rowsThatFit;
+	if (from <= -1) {
+		window.alert("Nee, stout >.<,your search has exceeded the minimum rows")
+		from = 0;
+		to = from + rowsThatFit;
+	}
+	fromValue = (<HTMLInputElement>document.getElementById("fromValue")).value = from.toString();
+	toValue = (<HTMLInputElement>document.getElementById("toValue")).value = to.toString();
+	callHeaders(from, to);
+
+}
+
+
+/**
+ *Retrieves column names
+*/
+function callHeaders(from: number, to: number) {
```

Comment:
This method is called _callHeaders_ and the next one is called _retrieveRows_.
Could you pls make the names of these functions a little bit more consistent throughout your code?